README.md

```diff
@@ -108,6 +109,18 @@ Executing `quickshot help` at any time will provide an overview of the available
   quickshot --help                 Show this screen.
 ```
 
+## Ignore file(s)
+
+After executing `quickshot configure` and selecting `Configure ignore file` you will be presented with the following menu:
+
+```
+? What would you like to use as the quickshot ignore file? (Use arrow keys)
+❯ .gitignore
+  .quickshotignore
+```  
+
+You would be wise to have `quickshot.json` in your `.gitignore` file but you may also want to ignore other files such as `settings_data.json` which contain data that could have been edited via Shopify's web admin panel that you don't want to overwrite using commands like `quickshot watch` (especially when working in multiple stores). 
```

Comment:
`quickshot.json` is always globally ignored. You dont need to add it to the ignore file.